examples/inbound-deliveries-to-sap/INBOUND DELIVERIES TO SAP.md

```diff
@@ -0,0 +1,50 @@
+# Extract delivery information from XML files and confirm receipt in S/4HANA inbound delivery
```

Comment:
```suggestion
# Integrate Warehouse System with S/4HANA
```

Reply:
Updated in [Update reviewed changes](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/16/commits/8650b99db9b8e7a6078a7120025cc0bf81979c17)